x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/execution/assembler/SequenceSpecTests.java

```diff
@@ -85,8 +86,8 @@ public String hitName() {
         static final TimestampExtractor INSTANCE = new TimestampExtractor();
 
         @Override
-        public Long extract(SearchHit hit) {
-            return (long) hit.docId();
+        public Object extract(SearchHit hit) {
```

Comment:
The return time should be Timestamp not Object.

Reply:
Fixed, thanks.